torch/csrc/utils/throughput_benchmark.h

```diff
@@ -0,0 +1,187 @@
+#pragma once
+
+#include <ATen/core/ivalue.h>
+#include <torch/csrc/jit/script/module.h>
+#include <pybind11/pybind11.h>
+
+#include <torch/csrc/jit/pybind_utils.h>
+
+#include <vector>
+#include <memory>
+
+namespace py = pybind11;
+
+namespace torch {
+namespace throughput_benchmark {
+
+/**
+ * The struct is used to provide results of a benchmark to the caller
+ * In the future all additional statics should be added here. This struct
+ * is exposed as an object to python with the following read-only properties:
+ *
+ * stats.qps
+ * stats.num_forwards
```

Comment:
Is this "num_iters"? Is "qps" still here?

Reply:
thanks, nice catch